app/controllers/recipes_controller.rb

```diff
@@ -22,6 +27,6 @@ def show
   private
 
   def recipe_params
-    params.require(:recipe).permit(:name, :style, :yeast, :summary, :notes)
+    params.require(:recipe).permit(:name, :style, :grains, :yeast, :summary, :notes)
```

Comment:
Once the form is working correctly when tested manually in the browser, and the recipe is successfully saved, then the feature spec should move past that error with `current_path` and `Recipe.first`.